spec/requests/casa_cases_spec.rb

```diff
@@ -111,6 +111,13 @@
           expect(casa_case.hearing_type).to eq hearing_type
           expect(casa_case.judge).to eq judge
         end
+
+        it "also respond to json", :aggregate_failures do
+          post casa_cases_url(format: :json), params: {casa_case: valid_attributes}
+
+          expect(response.content_type).to eq("application/json; charset=utf-8")
+          expect(response).to have_http_status(:created)
```

Comment:
I would like to also see an assert of the body contents here to make sure the fields come through correctly.